src/utils/getRandomTestData.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License.
```

Comment:
I might be missing the reason why we need to do this, but if possible it would be nice to avoid duplicating this code in multiple places.